client/ratings_reviews/helpers.js

```diff
@@ -139,6 +139,16 @@ const applyStarFilters = (reviews, starFilters) => {
   return output;
 };
 
+const applyKeyword = (reviews, keyword) => {
+  let output = reviews.filter(review => {
+    if (review.response === null) {
+      review.response = '';
+    }
+    return review.summary.includes(keyword) || review.body.includes(keyword) || review.reviewer_name.includes(keyword) || review.response.includes(keyword);
```

Comment:
This is cool--my search only looks at the text body of the "question". Maybe I should expand it to search everything!

Reply:
Honestly, it wasn't quite clear what we're supposed to be searching for exactly, so I think your interpretation is fine too!